packages/beacon-node/src/chain/archiveStore/historicalState/metrics.ts

```diff
@@ -66,6 +66,14 @@ export function createHistoricalStateTransitionMetrics(
       name: "lodestar_historical_state_stfn_num_effective_balance_updates_count",
       help: "Count of effective balance updates in epoch transition",
     }),
+    validatorsInActivationQueue: metricsRegister.gauge({
+      name: "lodestar_historical_state_stfn_validators_in_activation_queue_length",
+      help: "Total count of validators in the activation queue",
```

Comment:
we could rephrase this a bit I guess but it's a minor thing

```suggestion
      help: "Current number of validators in the activation queue",
```